### PR TITLE
Wider overlay container

### PR DESCRIPTION
**Pull request by akira28**
Related to Sulu PR https://github.com/sulu/sulu/pull/2514 and issue https://github.com/sulu/sulu/issues/2104

### CODE REVIEW

File: scss/variables.scss [hunk 1]
```diff
@@ -111,4 +111,4 @@ $navigationHoverColor: #191D22;
 // Overlay
 // -------------------------
 $overlayContainerWidth: 580px;
-$overlayContainerWidthExtraWide: 700px;
+$overlayContainerWidthExtraWide: 750px;
```
Comment (wachterjohannes) [inline scss/variables.scss]: we should introduce another skin for that (:

**Comment by danrot** [comment]
Do we need that change now? The linked PR has already been merged without that.

**Comment by akira28** [comment]
Is a styling issue, it depends on what you think about it, is not critical I suppose

**Comment by akira28** [comment]
This is how it looks without the change:

screenshot 2016-07-20 14 46 06

**Comment by danrot** [comment]
What's the problem here? Don't you like that 3 columns don't fit the width?

**Comment by akira28** [comment]
Exactly :D

**Comment by danrot** [comment]
@akira28 We just decided to make this overlay responsive, so that it takes as much as space as it has available. So we won't merge this PR, but you'll see that improvement quite soon on the develop branch :-)

**Pull request closed without merging**